src/test/java/com/google/javascript/gents/test_externs_map.json

```diff
@@ -0,0 +1,4 @@
+{
+  "jQuery": "JQuery"
```

Comment:
I wonder if it makes sense to hard code this, instead of configuring it? It's probably going to be the same stuff for most projects? OTOH, there might be specifics e.g. for internal Google use. Both works, I guess.